nestkernel/per_thread_bool_indicator.cpp

```diff
@@ -50,62 +50,47 @@ PerThreadBoolIndicator::initialize( const size_t num_threads, const bool status
   kernel().vp_manager.assert_single_threaded();
   per_thread_status_.clear();
   per_thread_status_.resize( num_threads, BoolIndicatorUInt64( status ) );
+  size_ = num_threads;
+  if ( status )
+    are_true_ = num_threads;
+  else
+    are_true_ = 0;
```

Comment:
We require braces also for single-statement blocks.
```suggestion
  if ( status )
  {
    are_true_ = num_threads;
  }
  else
  {
    are_true_ = 0;
  }
```